src/gui/mainwindow.cpp

```diff
@@ -2061,31 +2044,12 @@ void MainWindow::checkProgramUpdate()
 #endif
 
 #ifdef Q_OS_WIN
-bool MainWindow::addPythonPathToEnv()
```

Comment:
Where did this code go?

Reply:
Deleted, I not sure if it is really useful now.